.github/workflows/in_develop_labeler.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   label_preflight:
```

Comment:
```suggestion
  on-failure:
    runs-on: ubuntu-latest
    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
    steps:
      - run: echo "First workflow was a failure"
  label_preflight:
```

Reply:
Dunno how to link to the workflow run, but that should be possible too if needed